modules/client-configuration/pages/tshoot-clients.adoc

```diff
@@ -52,6 +52,45 @@ Therefore, we recommended that you do not power on bare metal systems multiple t
 
 
 
+== Bootstrap End-of-Life {centos} 6 Clients
+
+{centos} 6 is now at end-of-life, and the ISO images provided in the client repository for this operating system are out of date.
+Bootstrapping new {centos} 6 clients using these packages will fail.
+This failure will not happen on {centos} 6 clients that are already installed and bootstrapped.
+
+If you need to bootstrap new {centos} 6 clients you can remove the old repositories, and create a new repository to provide the correct packages.
+
+
+
+.Procedure: Troubleshooting Bootstrapping a New {centos} 6 Client
+. On the {productname} Server, at the command prompt, remove the repositories provided by the {centos} 6 ISO:
++
+----
+rm CentOS-Base.repo CentOS-Debuginfo.repo CentOS-fasttrack.repo
+----
+. Add a new repository:
++
+----
+zypper addrepo -f http://vault.centos.org/6.10/os/$basearch/
```

Comment:
CentOS does not have zypper. The files need to be written manually.
I would suggest we do not describe to remove them, but just change the URL to vault.

Reply:
Updated.